ArcGISRuntimeSDKQt_CppSamples/Maps/DisplayOverviewMap/DisplayOverviewMap.qml

```diff
@@ -0,0 +1,60 @@
+// [WriteFile Name=DisplayOverviewMap, Category=Maps]
+// [Legal]
+// Copyright 2021 Esri.
+
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+// http://www.apache.org/licenses/LICENSE-2.0
+
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// [Legal]
+
+import QtQuick 2.12
+import QtQuick.Controls 2.12
+import Esri.Samples 1.0
+import Esri.ArcGISRuntime.Toolkit 100.13
+
+Item {
+
+    // add a mapView component
+    MapView {
+        id: view
+        anchors.fill: parent
+
+        Rectangle {
+            id: overviewMapBorder
+            width: view.width * 0.4
+            height: view.height * 0.35
+            anchors {
+                left: view.left
+                leftMargin: 5
+                bottom: view.attributionTop
+                bottomMargin: 5
+            }
+            color: "transparent"
+            border.color: "black"
+            border.width: 2
+
+            OverviewMap {
+                id: insetMap
```

Comment:
maybe just call it `overviewMap` rather than having another term for the same thing